src/site/content/en/learn/css/media-queries/index.md

```diff
@@ -0,0 +1,786 @@
+
+CSS styles are applied to documents that can be consumed in a medium other than browsers; accordingly media queries are a handy way for Web creatives to conditionally apply styles to fix, enhance, or maximize a document's usefulness accounting for particular media conditions applicable
+
+In other words, keeping in mind `media` is plural for medium, CSS media queries allow you to apply styles originating from blocks of CSS or a stylesheet depending on medium-related conditions applicable to the device a user is viewing a particular document with. These media-related conditions include the capabilities you require a medium to have & even the users preferences being actively accommodated by the medium.
+
+In CSS you apply a media query using either the  `@media`  or `@import`  [at rules](https://thecsspodcast.libsyn.com/031-rules)  followed by the [**media query**](l).
+
+`@media` is used to conditionally apply styles based on a media query while `@link` conditionally loads a stylesheet based on a media query.
+
+Media queries immediately follow the `@media`   at-rule, while media queries follows the URL to a stylesheet & optionally feature queries when used with the `@import` at-rule.
+
+To begin the module we conditionally placed cards on a grid with the following code:
+
+```css
+#cards {
+  display: grid;
+  grid-template: 1fr;
+  grid-gap: 2rem;
+  width: 100%;
+  padding: 10%;
+  margin: 0;
+  list-style: none;
+  border: 2px solid magenta;
+}
+
+@media (min-width: 31.25em /* 500px */)  {
+  #cards {
+    grid-template-columns: 1fr 1fr;
+  }
+}
+
+@media (min-width: 65.8125em /* 1053px */)  {
+  #cards {
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+@media(min-width: 80em /* 1280px */) {
+  #cards {
+    grid-template-columns: repeat(auto-fill, minmax(20em, 1fr))
+  }
+}
+```
+
+The equivalent if the `@media` at-rule code blocks were co-located in a separate stylesheet  would be the following
+
+```css
+@import "meaningful-layout-tweaks-to-cards-on-larger-viewports.css" (min-width: 31.25em);
+```
+
+{% Aside 'warning' %}
+Because of performance reasons, `@import` should be utilized with caution. It often is a better choice to use `link` or concatenating the styles you considered importing instead.
+{% endAside %}
+
+With all of this in mind, the following section goes over the constructs that make up a media query.
+
+## The anatomy of a Media Query
+A  media query is intrinsically a *media condition* consisting of the following:
+
+* An optional query modifier to reverse the meaning of a query expression using the `not` keyword or to restrict styles against legacy media contexts  using the `only` keyword
+
+*  A conditionally optional *media type*
```

Comment:
```suggestion
* An optional query modifier to reverse the meaning of a query expression using the `not` keyword or to restrict styles against legacy media contexts  using the `only` keyword
*  A conditionally optional *media type*
```

Reply:
IIRC, markdown lists shouldn't have two line breaks, but it may not actually matter.